blog/2023-02-03-tauri-2-0-0-alpha-3 .mdx

```diff
@@ -0,0 +1,40 @@
+---
+authors: [wusyong]
+description: New v2.0 alpha released! New backend for Linux port!
+---
+
+# Migration to `webkit2gtk-4.1` on Linux port!
+
+Hello everybody! We just release another alpha version lately. While it doesn't bring any major features, it does bring some huge impact on Linux port.
+We will use [WebKit2GTK–4.1](https://webkitgtk.org/reference/webkit2gtk/stable/index.html) in 2.0 from now on.
+
+<!--truncate-->
+
+## What does this mean?
+
+If you are using Tauri version 1.x, there's nothing to worry about. Everything you need is still the same.
+But if you are using Tauri version 2.0 alpha version starting from `alpha.3`, you will need to install new WebKit2GTK package with API version 4.1.
+We will update Prerequisites in our site soon. But if you want to know how to install such version, here are some instructions from [wry](https://github.com/tauri-apps/wry#linux):
+
+```bash
+# On Arch Linux / Manjaro:
+sudo pacman -S webkit2gtk-4.1
+# On Debian / Ubuntu:
+sudo apt install libwebkit2gtk-4.1-dev
+# On Fedora:
+sudo dnf install webkit2gtk4.1-devel
+```
+
+## Will this bring breaking changes to my code?
+
+The main difference between version 4.0 and 4.1 are the soup library they use 4.0 uses soup2 and 4.1 uses soup3.
+So if you didn't use any soup2 specific APIs, you applications should be fine.
+
+The reason we want to make this change is because we aim to add flatpak support, but the WebKitGTK [Gnome runtime](https://docs.flatpak.org/en/latest/available-runtimes.html#gnome) uses is webkit2gtk-4.1.
+There are also some subtle bugs like [this](https://github.com/tauri-apps/wry/issues/717) only happen in soup2 and they can be fixed by upgrading to soup3.
+
+## What else breaking changes we are going to expect?
+
+The major one will be the MSRV. With Tauri v2.0-alpha.3 released, its MSRV is bumped to 1.64. We are also going to update windows-rs in the future.
+But this rust version should satisfy latest version of windows-rs unless there is a newer version coming out.
+We do plan to update our MSRV with minor release after 2.0. This could ease the tension to stick to any fixed rust version while updating dependencies.
```

Comment:
```suggestion
We do plan to update our MSRV with minor releases after 2.0. This could ease the friction to stick to any fixed Rust version while updating dependencies.
```